### DISCUSSION TITLE
version challenges

### DISCUSSION THREAD

**Discussion by DarthMuppet54** [itzg/docker-minecraft-server#1737] [Q&A] [answered]
Running this in Truenas using K3S. I had the image tag set to 'latest' and rebooted the host to move it. After this, the user (my son) tells me he wants to stick on 1.19, not 1.19.2.  no problem I thought, the server is Paper, I just added a version variable, set to 1.19, it downloads it and launches it.

When the server starts the log shows 'java.lang.RuntimeException: Server attempted to load chunk saved with newer version of minecraft! 3120 > 3105' when loading the nether. No problem, I restore the last backup from before the reboot, and that error is resolved. now it tells me the same thing loading the_end. I just delete that folder, it recreates it then gives exactly the same error. 

With everything on 1.19.2 it all works, but he insists he insists he needs 1.19.  I have backups of the entire data folder going back aeons, so reverting to the last functioning date is fine, I just need to know how to get the container back to the previous version. What information do I need to work that out?  The image tag I'm using is currently 2022.11.0, but latest has the same issue. The tags and versions mean nothing to me unfortunately so i have no idea which ones to try next. Last time the container was updated would have been June.  I am not a docker expert, so if I need to do anything there beyond getting container logs, you'll need to tell me how to do it. I am very much a techie, but this is all new to me. 

Thanks.

**Comment by itzg** [comment, marked as answer]
The image version is completely independent of the server/Paper version. 

Setting VERSION=1.19 is the answer and should have been set in the first place if that was important. 

Downgrading with any software is risky, so going from 1.19.2 to 1.19 is not an unusual risk. Perhaps search around for any advice or downgrade tooling from Paper community.

**Comment by DarthMuppet54** [comment]
Good to know. I get that downgrading is not going to work, hence restoring, but what actually threw me was that the server and world had already updated to 1.19.2 way before I was told they had.  I just go back to a real 1.19 world and everything's fine. Also back on latest for the container. 

Thanks, appreciate you taking the time.
